pages.zh/common/docker-build.md

```diff
@@ -0,0 +1,32 @@
+# docker build
+
+> 从 Dockerfile 打包镜像。
+> 更多信息：<https://docs.docker.com/engine/reference/commandline/build/>.
+
+- 使用当前目录下的 Dockerfile 打包一个 docker 镜像：
+
+`docker build .`
+
+- 从指定 URL 的 Dockerfile 打包 docker 镜像：
+
+`docker build {{github.com/creack/docker-firefox}}`
+
+- 打包一个 docker 镜像并指定镜像的标签：
+
+`docker build --tag {{name:tag}} .`
+
+- 打包一个没有上下文的 docker 镜像：
+
+`docker build --tag {{name:tag}} - < {{Dockerfile}}`
+
+- 打包镜像时不使用缓存：
+
+`docker build --no-cache --tag {{name:tag}} .`
```

Comment:
Could the {{name:tag}} not be translated?

Reply:
Yes, this is a tutorial for translating Chinese.

I personally feel that {name:tag} is better understood if it is not translated, and of course it can be translated, but the developer may look a bit awkward.

FYI @CleanMachine1 @navarroaxel

Reply:
Because in the previous line of the command, the semantics of the command parameters have been clearly stated, so I think it is clear enough.

Of course, the opinions of the two members are also very important, thank you for taking the time to Review and submit.